test/api_integration/apis/saved_objects/import.js

```diff
@@ -112,6 +112,30 @@ export default function ({ getService }) {
             });
         });
 
+        it('should return 200 when trying to import unsupported types', async () => {
+          const fileBuffer = Buffer.from('{"id":"1","type":"wigwags","attributes":{"title":"my title"},"references":[]}', 'utf8');
+          await supertest
+            .post('/api/saved_objects/_import')
+            .attach('file', fileBuffer, 'export.ndjson')
+            .expect(200)
```

Comment:
Why is this a 200 and not a 400?

Reply:
The import and resolve import errors API follow the elasticsearch bulk API style where it always returns 200 but you need to make sure no errors are returned in the response.